docs/build/build-options.md

```diff
@@ -111,113 +235,133 @@ This option allows control through the `CMAKE_BUILD_TYPE` variable:
 development when testing on a local network.
 
 
+#### ENABLE_HAICRYPT_LOGGING
 **`--enable-haicrypt-logging`** (default: OFF)
 
-Enables logging in the *haicrypt* module, which serves as a connector to
+When ON, enables logging in the *haicrypt* module, which serves as a connector to
 an encryption library. Logging here might be seen as unsafe, therefore this 
 option is turned OFF by default.
 
 
+[:arrow_up: &nbsp; Back to List of Build Options](#list-of-build-options)
+
+
+#### ENABLE_HEAVY_LOGGING
 **`--enable-heavy-logging`** (default: OFF in release mode)
 
-This option enables the logging instructions in the code, which are considered
+When ON, this option enables logging instructions in the code, which are considered
 heavy as they occur often and cover many detailed aspects of library behavior.
 Turning this option ON will allow you to use the `debug` level of logging and get 
 detailed information as to what happens inside the library. Note, however, that 
-this may influence processing by changing times, using less preferred thread 
+this may influence processing by changing timings, use less preferred thread 
 switching layouts, and generally worsen the functionality and performance of 
 the library. For these reasons this option is turned OFF by default.
 
 
+#### ENABLE_INET_PTON
 **`--enable-inet-pton`** (default: ON)
 
-Enables usage of `inet_pton` function by the applications, which should be used
-to resolve the network endpoint name into an IP address. This may be not
-availabe on some version of Windows, in which case you can turn this OFF.
-When this option is OFF, however, IP addresses cannot be resolved by name,
-as the `inet_pton` function gets a poor-man's simple replacement that can
+When ON, enables usage of the `inet_pton` function by applications, which should 
+be used to resolve the network endpoint name into an IP address. This may not be 
+available in some versions of Microsoft Windows, in which case you can change the 
+setting to OFF. When this option is OFF, however, IP addresses cannot be resolved 
+by name, as the `inet_pton` function gets a poor-man's simple replacement that can
 only resolve numeric IPv4 addresses.
 
 
+#### ENABLE_LOGGING
 **`--enable-logging`** (default: ON)
 
-Enables logging. When you turn this option OFF, the library will not report
-any runtime information through the logging system, including errors.
-This option may be useful if you suspect the logging system of
-impairing performance.
+When ON, enables logging. When you turn this option OFF, the library will not 
+report any runtime information, including errors, through the logging system. This 
+option may be useful if you suspect the logging system of impairing performance.
 
 
+#### ENABLE_MONOTONIC_CLOCK
 **`--enable-monotonic-clock`** (default: OFF)
 
-This option enforces the use of `clock_gettime` to get the current
+When ON, this option enforces the use of `clock_gettime` to get the current
```

Comment:
```suggestion
**NOTE**: The library can get stuck if the system clock is used instead of 
monotonic or C++11 steady. Since v1.4.4 `ENABLE_MONOTONIC_CLOCK` is enabled by 
default on POSIX-type systems if support for CLOCK_MONOTONIC is detected. 
On Windows `ENABLE_STDCXX_SYNC` is enabled by default. It is highly recommended 
to use either of those (`ENABLE_STDCXX_SYNC` excludes `ENABLE_MONOTONIC_CLOCK`).

When ON, this option enforces the use of `clock_gettime` to get the current
```